ldes-consumer/app.py

```
@@ -0,0 +1,591 @@
+    logger.info("=" * 60)
+    logger.info("LDES Consumer Application")
+    logger.info("=" * 60)
+    logger.info(f"Reading configuration from: {config_path}")
+    logger.info(f"LDES2SPARQL Image: {ldes2sparql_image}")
+
+    # Initialize Docker client
+    try:
+        docker_client = docker.from_env()
+        docker_client.ping()
+        logger.info("✓ Docker connection established")
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)